tests/test_inventory_builder.py

```diff
@@ -35,6 +37,29 @@ def config():
     return conf
 
 
+@pytest.fixture
```

Comment:
Here a simple dict would be enough. No need for instanciating an Inventory Builder object since it will only return a dict that is present in the configuration file content.